omod/src/main/resources/config.xml

```diff
@@ -88,6 +88,72 @@
         </description>
     </globalProperty>
 
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpiSimilarPatientSearchAlgorithm</property>
+		<defaultValue>registrationcore.OpenEmpiExactPatientSearchAlgorithm</defaultValue>
+		<description>
+			Specifies a bean used for the precise similar patient search algorithm.
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.globalIdentifierDomainId</property>
+		<defaultValue></defaultValue>
+		<description>
+			Specifies main identifier on MPI server
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.implementation</property>
+		<defaultValue></defaultValue>
+		<description>
+			Specifies a bean name of MPI implementation
```

Comment:
"Which MPI to should we connect to? Specify a Spring bean name, or leave blank to disable MPI integration"